docs/src/@primer/gatsby-theme-doctocat/components/theme-switcher.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import {ThemeProvider, useTheme /* Dropdown*/} from '@primer/components'
+
+function ThemeSwitcherProvider({children, colorProps}) {
+  return <ThemeProvider {...colorProps}>{children}</ThemeProvider>
+}
+
+function ThemeSwitcher({onSwitch}) {
+  const {theme} = useTheme()
+  //   return <Dropdown>
+  //   <Dropdown.Button>ColorScheme</Dropdown.Button>
+  //   <Dropdown.Menu
+  //     direction="sw"
+  //     items={theme.colorSchemes}
+  //     onChange={colorScheme => {
+  //       const colorMode = colorScheme === 'light' ? 'day' : 'night'
+  //       const nightScheme = colorScheme === 'light' ? 'dark' : colorScheme
+  //       onSwitch({
+  //         colorMode,
+  //         dayScheme: 'light',
+  //         nightScheme
+  //       })
+  //     }}
+  //   ></Dropdown.Menu>
+  // </Dropdown>
+  return (
+    <select
+      style={{float: 'right'}} // sorry
+      onBlur={event => {
```

Comment:
Hi hi hi!

Should we use `onChange` instead of `onBlur` so that it changes instantly. (It does change in your demo gif though!

This is what I see on Chrome:
![Kapture 2021-09-27 at 14 01 38](https://user-images.githubusercontent.com/1863771/134904358-36e06779-3c68-4914-a639-0f0166169d5b.gif)
)


```suggestion
      onChange={event => {
```

Reply:
oops sorry! There was a wierd eslint rule on `<select />` suggesting me to change to `onBlur` I remember. Anyway, this shouldn't be an issue with the `DropdownMenu` component.